webapp/angular.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "$schema": "../node_modules/@angular/cli/lib/config/schema.json",
```

Comment:
Did that move from where it was before, or was this misconfigured and now fixed?